bucket/ansicon.json

```
@@ -0,0 +1,23 @@
+{
+    "homepage": "http://ansicon.adoxa.vze.com/",
+    "description": "Process ANSI escape sequences for Windows console programs.",
+    "license": "https://github.com/adoxa/ansicon/blob/master/LICENSE.txt",
```

https://opensource.org/licenses/Zlib
https://spdx.org/licenses/Zlib.html

```suggestion
    "license": "zlib",
```